### ISSUE TITLE
dumbPaintTool doesn't start properly?

### ISSUE THREAD

**Issue by BMaxV** [ceccopierangiolieugenio/pyTermTk#266]
Hi,

I tried installing dumbPaintTool and it doesn't work? 

I installed with pip.

```
Traceback (most recent call last):
  File "/home/max/.local/bin/dumbPaintTool", line 5, in <module>
    from dumbPaintTool import main
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/__init__.py", line 23, in <module>
    from .app import main
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/app/__init__.py", line 1, in <module>
    from .main         import *
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/app/main.py", line 30, in <module>
    from .maintemplate import PaintTemplate
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/app/maintemplate.py", line 29, in <module>
    from .paintarea    import PaintArea, PaintScrollArea
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/app/paintarea.py", line 29, in <module>
    from .glbls       import glbls
  File "/home/max/.local/lib/python3.10/site-packages/dumbPaintTool/app/glbls.py", line 29, in <module>
    from .state.brush  import Brush
ModuleNotFoundError: No module named 'dumbPaintTool.app.state'

```

**Comment by ceccopierangiolieugenio** [comment]
That's very strange,
Which is your setup?

**Comment by ceccopierangiolieugenio** [comment]
Fix delivered in "**dumbPaintTool-0.40.0a4**"
